docs/Upgrade-Guide/upgrade_to_3.12.md

```diff
@@ -84,7 +91,12 @@ This procedure involves cluster downtime and during the upgrade window, clients
     #gluster volume status
 ```
 
-7. Restart any gfapi based application stopped previously in step (2)
+7. Start glustereventsd service (if it was enabled before), either using the command below, or through other means,
```

Comment:
If the glustereventsd service was previously enabled, it is required to start it using the commands below, or, through other means

Reply:
Done